from app.exceptions.repositories.base import BaseRepoError


class ConnectionRefusedRepoError(BaseRepoError):
    pass
